### PR TITLE
added last updated to practitionerRole and updated README.

### PR DESCRIPTION
**Pull request by bcastillomartel-va**
lastUpdated for practitionerRole resource

### CODE REVIEW

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/MinimartMaker.java [hunk 1]
```diff
@@ -527,8 +528,7 @@ public static void sync(String directory, String resourceToSync, String configFi
 
   private void cleanUpEntityManagers() {
     /*
-     * Commit and clean up the transactions for the entity managers from
-     * the various threads.
+     * Commit and clean up the transactions for the entity managers from the various threads.
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/MinimartMaker.java:531]: Was this intentional? Run the formatter on the command-line and recommit

Comment (bcastillomartel-va) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/MinimartMaker.java:531]: I did run `mvn clean install`, let me try again

File: README.md [hunk 2]
```diff
@@ -13,10 +13,16 @@ See also
 2. Download [Flyway](https://flywaydb.org/download/) and unzip it in the root of this repository
 3. Use `./run-local.sh clean` to create a new database (as a docker image) and populate it
 
-To repopulate an existing database, retaining existing schema and records, use `./run-local.sh`
+- To repopulate an existing database, retaining existing schema and records, use `./run-local.sh`
```
Comment (taylorrassmannlibertyits) [inline README.md:16]: I'm confused why this shows up as a change? Just a question

Comment (taylorrassmannlibertyits) [reply README.md:16]: Ah I see you added a `-`

File: minimart-manager/README.md [hunk 3]
```diff
@@ -31,4 +31,4 @@ mvn exec:java -Dexec.mainClass="gov.va.api.health.minimartmanager.augment.IdVali
     - This process is also auto-triggered when lab [health-apis-datamart-synthetic-records](https://github.com/department-of-veterans-affairs/health-apis-datamart-synthetic-records/tree/lab) is built
 - `export-h2`
   - Export a minimart instance to an H2 database at `target/mitre.mv.db`
-  - `config/lab.properties` is required with connection properties for the source database
\ No newline at end of file
+  - `config/lab.properties` is required with connection properties for the source database
```
Comment (taylorrassmannlibertyits) [inline minimart-manager/README.md:34]: what changed on this line?

**Review by jblefkove-va — CHANGES_REQUESTED** [review]
Move documentation down

File: README.md [hunk 4]
```diff
@@ -13,10 +13,10 @@ See also
...
```
Comment (jblefkove-va) [inline README.md:16]: Now that the other documentation has moved, I don't think these bullets are necessary

File: minimart-manager/README.md [hunk 5]
```diff
@@ -31,4 +31,12 @@ mvn exec:java -Dexec.mainClass="gov.va.api.health.minimartmanager.augment.IdVali
     - This process is also auto-triggered when lab [health-apis-datamart-synthetic-records](https://github.com/department-of-veterans-affairs/health-apis-datamart-synthetic-records/tree/lab) is built
 - `export-h2`
   - Export a minimart instance to an H2 database at `target/mitre.mv.db`
-  - `config/lab.properties` is required with connection properties for the source database
\ No newline at end of file
+  - `sqlserver.properties` is required with connection properties for the source database
+  
+## Local Development
+
+- Whenever there are changes to the `mitre.mv.db` file, complete the following steps:
```
Comment (jblefkove-va) [inline minimart-manager/README.md:38]: This would have more context with a link to https://github.com/department-of-veterans-affairs/health-apis-data-query/blob/master/data-query-tests/src/test/resources/mitre.mv.db

File: minimart-manager/README.md [hunk 6]
```diff
@@ -31,4 +31,13 @@ mvn exec:java -Dexec.mainClass="gov.va.api.health.minimartmanager.augment.IdVali
...
   - Export a minimart instance to an H2 database at `target/mitre.mv.db`
-  - `config/lab.properties` is required with connection properties for the source database
\ No newline at end of file
+  - `sqlserver.properties` is required with connection properties for the source database
+  
+## Local Development
+
+- Whenever there are changes to the [`mitre.mv.db`](https://github.com/department-of-veterans-affairs/health-apis-data-query/blob/master/data-query-tests/src/test/resources/mitre.mv.db) file, complete the following steps:
+  - `cd ..`
+  - `Update docker image `./run-local.sh start`
```
Comment (jblefkove-va) [inline minimart-manager/README.md:40]: I am going to extract these steps into their own script

Comment (bcastillomartel-va) [reply minimart-manager/README.md]: that's a major improvement, neat
